src/tools/common.ts

```diff
@@ -174,3 +174,28 @@ export const chooseFile: ToolFactory = snapshot => ({
     }, snapshot);
   },
 });
+
+const resizeSchema = z.object({
```

Comment:
Let them specify pixels? We should be careful about exposing it as it is niche and unclear how it is going to be used. Original request wanted it for responsive design where 800 vs 1024 vs 1280 could result in different media query matches...

Reply:
done in https://github.com/microsoft/playwright-mcp/pull/92/commits/5699c1703e51a5739f9b826e301a2c49e83eb00c